atoma-service/src/streamer.rs

```diff
@@ -33,15 +34,34 @@ use crate::{
 
 /// The chunk that indicates the end of a streaming response
 const DONE_CHUNK: &str = "[DONE]";
+
 /// The prefix for the data chunk
 const DATA_PREFIX: &str = "data: ";
+
 /// The keep-alive chunk
 const KEEP_ALIVE_CHUNK: &[u8] = b": keep-alive\n\n";
+
 /// The choices key
 const CHOICES: &str = "choices";
+
 /// The usage key
 const USAGE: &str = "usage";
 
+/// The ciphertext key
+const CIPHERTEXT_KEY: &str = "ciphertext";
+
+/// The nonce key
+const NONCE_KEY: &str = "nonce";
+
+/// The response hash key
+const RESPONSE_HASH_KEY: &str = "response_hash";
+
+/// The signature key
+const SIGNATURE_KEY: &str = "signature";
+
+/// The usage key
+const USAGE_KEY: &str = "usage";
```

Comment:
You have the same keys in the atoma-service/src/handlers/mod.rs, maybe move it to a new file and use from there.